modules/nf-core/sentieon/README.md

```diff
@@ -0,0 +1,18 @@
+# Updating the docker container and making a new module release
```

Comment:
The README-file is much appreciated 👏

Reply:
🌊 I think wave will also make this a non-issue? Maybe I'm too eager.

Reply:
yeah, but we're not there yet